pkgs/development/python-modules/reproject/default.nix

```diff
@@ -0,0 +1,29 @@
+{ pkgs
+, buildPythonPackage
+, fetchPypi
+, numpy
+, astropy
+, astropy-healpix
+, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "reproject";
+  version = "0.4";
+
+  doCheck = false; # the tests requires several pytest plugins that are not in nixpkgs
```

Comment:
Which ones?

Reply:
`pytest-astropy` and its dependencies (`pytest-doctestplus`, `pytest-remotedata`, `pytest-openfiles` and `pytest-arraydiff`).